mlrun/common/schemas/client_spec.py

```diff
@@ -67,3 +67,4 @@ class ClientSpec(pydantic.v1.BaseModel):
     alerts_mode: typing.Optional[str]
     system_id: typing.Optional[str]
     model_endpoint_monitoring_store_prefixes: typing.Optional[dict[str, str]]
+    httpdb_authentication_mode: typing.Optional[str]
```

Comment:
`httpdb` prefix no really needed imo
```suggestion
    authentication_mode: typing.Optional[str]
```